### ISSUE TITLE
Cant get android running

### ISSUE THREAD

**Issue by vongohren** [bugsnag/bugsnag-react-native#97]
Im struggeling with android, im trying to do the manual setup.
```
include ':bugsnag-react-native'
project(':bugsnag-react-native').projectDir = new File(rootProject.projectDir, '../node_modules/bugsnag-react-native/android')
```

`compile project(':bugsnag-react-native')`

Im adding it to the settings and build gradle, then initiate in the MainActivit class as noted i setup.
But the project doesnt even build because

`Gradle sync failed: Cannot read packageName from /Users/snorreedwin/Code/entur/entur-clients/native/node_modules/bugsnag-react-native/android/src/main/AndroidManifest.xml`

I tried running the example app, sync and build it, but it failes on 

`Error:(1, 0) Plugin with id 'com.android.library' not found.
<a href="openFile:bugsnag-react-native/android/build.gradle">Open File</a>`

**Comment by kattrali** [comment]
Thanks for the report, @vongohren. What versions of react-native and bugsnag-react-native are you using? 

The latest version of bugsnag-react-native:
* for react-native <= 0.39 is 1.2.2
* for react-native >= 0.40 is 2.2.3

**Comment by vongohren** [comment]
Im using 2.2.3. But i have to say I left this library. Because my main problem was that the android sourcemaps from react-native is not properly deduced. So, now im currently manually doign that jobb.

**Comment by kattrali** [comment]
I think this is probably a caching issue if you're using v2.2.3, as this exact error was fixed in #96. Can you try running `./gradlew clean` from the `android` directory of your project?

**Comment by vongohren** [comment]
Iv removed it from my project before you replied so I cant run it, but now there is an issue on it that people can go back and look at. Thanks for your answere
